migration/v0.16.x-v0.17.x/migrate-openid.sh

```diff
@@ -10,8 +10,8 @@ set -euo pipefail
 "${here}/../../bin/ck8s" ops kubectl sc delete pod -n elastic-system -l role=master
 
 # Wait for master pod to be ready before updating the config
-"${here}/../../bin/ck8s" ops kubectl sc wait --for=condition=ready pod  -n elastic-system opendistro-es-master-0
-sleep 10
+"${here}/../../bin/ck8s" ops kubectl sc wait --for=condition=Ready pod -n elastic-system opendistro-es-master-0
```

Comment:
Haven't looked at the kubectl code but I'm fairly certain that `ready==Ready` in this case.